tests/snippets/slice_res.py

```diff
@@ -0,0 +1,2252 @@
+SLICES_RES = [
```

Comment:
Could this be added to a .gitignore, if it's automatically generated during testing? Also, why does this need to be generated as a file? Why can't you just use a generated list?

Reply:
No the file is generated by cpython to compare cpython results with rustpython

Reply:
but indeed we can generate it on the fly with a subprocess.checkouput. i'll do it

Reply:
I was thinking we could generate it on the fly wichh a subprocess.chekoutput but it's not yet implemented so we have to build it before manually.
Maybe adding a fixture/resources directory in tests to put this kind of file

Reply:
How about building it in `tests_snippets.py` test `setUpClass`, then simply try `import`ing the generated file in one of the snippets?

Reply:
I was testing with cargo run tests/snippets/builtin_slice.py in compraison with vith python tests/snippets/builtin_slice.py. I don't know how to  use test_snippet.py with it.

Reply:
> I don't know how to use test_snippet.py with it.

`pytest -k builtin_slice` will run (and test setup steps) and test `builtin_slice.py`  for both CPython and RPy.

Reply:
ah ok thanks.
you mean something like that ? :
```python
def generate_slices():
    a = ["some things"]
    return "SLICE_RES={}".format(a)


@populate('cpython')
# @populate('cpython_bytecode')
@populate('rustpython')
class SampleTestCase(unittest.TestCase):
    @classmethod
    def setUpClass(cls):
        from pathlib import Path
        cls.p = Path(__file__).parent /"snippets"/'some_name_for_slices_res_file.py'
        cls.p.write_text(generate_slices())

    @classmethod
    def tearDownClass(cls):
        cls.p.unlink()
```
since slice_res.py should never change, If find that the previous code add some useless complexity
I really a think a submodule `ressource` inside snippets could be fine and simple.

Reply:
Yeah, something like that.
> I really a think a submodule ressource inside snippets could be fine and simple.
That's fine too. Personally I'd prefer it to be autogenerated (makes repo and searching code simpler; also the generated code could live in `resources` too), but I don't have a strong opinion on that.

Reply:
done